src/oic/utils/aes.py

```diff
@@ -16,7 +17,11 @@
 
 __author__ = "rolandh"
 
-POSTFIX_MODE = {"cbc": AES.MODE_CBC, "cfb": AES.MODE_CFB, "ecb": AES.MODE_CFB}
+POSTFIX_MODE: dict[str, Union[Literal[2], Literal[3]]] = {
```

Comment:
seems it should be typing.Dict[] to work, according to the test output.

Reply:
Yes, that is python 3.9 only :/ Fixed.